test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -2656,8 +2656,8 @@ BOOST_AUTO_TEST_CASE(value_for_constructor)
 		}
 		contract Main {
 			Helper h;
-			function Main() payable {
-				h = (new Helper).value(10)("abc", true);
+			constructor() payable {
+				h = (new Helper).value(10)('abc', true);
```

Comment:
Any reason for this change?

Reply:
Oh, this was an unconscious change due to broken code highlighting in my editor. We can revert to ``"abc"``